tests/Blocks/BlocksExampleTest.php

```diff
@@ -218,9 +218,13 @@
 test('Asserts that renderWrapperView will return a valid file.', function () {
 	$wrapperManifest = dirname(__FILE__, 2) . '/data/src/Blocks/wrapper/wrapper.php';
 
+	ob_start();
 	$this->blocksExample->renderWrapperView($wrapperManifest, []);
+	$content = ob_get_clean();
 
-	$this->assertTrue(true);
+	$this->assertSame('
+<div>Wrapper!</div>
+', $content);
```

Comment:
How about this:

```suggestion
	$this->assertSame('<div>Wrapper!</div>', trim($content));
```

A BIT less safe but much more pretty.

Reply:
Nope, Doesn't work, because the `renderWrapperView` will actually output the HTML to the OB. So all the break lines will be preserved. 

I added it like this the first time and the test failed 😂